### ISSUE TITLE
sankaku stops downloading pictures after page 58

### ISSUE THREAD

**Issue by naruhina2016** [Nandaka/DanbooruDownloader#191]
the title says it all.
i've already setup the cookie account thing.
please help me.

**Comment by Nandaka** [comment]
try this release? https://github.com/Nandaka/DanbooruDownloader/releases/tag/v3.20191125

**Comment by naruhina2016** [comment]
no still no success...:(

**Comment by Nandaka** [comment]
can you upload the logging?

**Comment by naruhina2016** [comment]
you mean this one?
[log-2019-11-25.txt](https://github.com/Nandaka/DanbooruDownloader/files/3884269/log-2019-11-25.txt)

**Comment by Nandaka** [comment]
Hmm, it is working in my side (tested with `https://chan.sankakucomplex.com/?tags=toujou_nozomi&page=59&next=`). 

Ensure the user agent is the same with your browser user agent used for login?
![image](https://user-images.githubusercontent.com/836867/69504177-75bd4400-0f5b-11ea-9a48-3eff65fb4c6b.png)

**Comment by naruhina2016** [comment]
you mean the version of my browser?

**Comment by Nandaka** [comment]
no, the browser user agent string, try this one: https://www.google.com/search?q=what+is+my+useragent

**Comment by naruhina2016** [comment]
its not working...:(

**Comment by Nandaka** [comment]
hmm, weird. Can you upload your `DanbooruProviderList.xml` screen shot? The cookie values should be pasted in the username section?
![image](https://user-images.githubusercontent.com/836867/69506369-431a4800-0f69-11ea-8b66-acf80b8ee298.png)

**Comment by naruhina2016** [comment]
here you go

![Untitled1](https://user-images.githubusercontent.com/18094007/69507108-b43e0900-0f31-11ea-810f-a2422e004c49.png)

**Comment by Nandaka** [comment]
can you copy the whole cookie string from the request header and paste it to the username section, not only the login/pass_hash/_sankakucomplex_session and retry?

1. login to you account
2. press F12 and navigate to my account
3. Copy the cookies and user agent from here and paste it to the `DanbooruProviderList.xml`
![image](https://user-images.githubusercontent.com/836867/69509927-d0fc3000-0f75-11ea-9b21-c9debc8da76d.png)

**Comment by Nandaka** [comment]
modified on cookie handling, replace the old exe
[DanbooruDownloader3.exeonly.zip](https://github.com/Nandaka/DanbooruDownloader/files/3884767/DanbooruDownloader3.exeonly.zip)

**Comment by naruhina2016** [comment]
but where do i have to place them in DanbooruProviderList.xml?

**Comment by Nandaka** [comment]
The cookies? just paste it as single line in the username section of sankaku (see the screenshot https://github.com/Nandaka/DanbooruDownloader/issues/191#issuecomment-557956890)

**Comment by naruhina2016** [comment]
in that file right? and what about UserAgent?

**Comment by Nandaka** [comment]
it should be in the settings tab (https://github.com/Nandaka/DanbooruDownloader/issues/191#issuecomment-557944467)

**Comment by naruhina2016** [comment]
but the cookie where in DanbooruProvideerList am i suppose to place it?

**Comment by Nandaka** [comment]
yep, cookie inside the DanbooruProvideerList.xml, user agent on the Settings Tab

**Comment by naruhina2016** [comment]
i cant seems to paste the cookie

**Comment by Nandaka** [comment]
use notepad or notepad++ (https://notepad-plus-plus.org/downloads/)

**Comment by naruhina2016** [comment]
its not working did everything you told me to do and yet it did not work....:(

**Comment by Nandaka** [comment]
can you try it on different pc/different network? because it is working on win10 pc (max 100pages because it is limited from the server).

**Comment by naruhina2016** [comment]
no i have only one desktop/laptop and my network is wired cant go wireless

**Comment by Nandaka** [comment]
No, I mean try to run it for different PC to check if it is something from your pc that cause the issue. Alternatively, you can try to use https://github.com/Bionus/imgbrd-grabber

**Comment by naruhina2016** [comment]
is image grabber better?

**Comment by Nandaka** [comment]
not sure, but looks like it have more support/developer. Can you get more than 58pages for sankaku using that?

**Comment by naruhina2016** [comment]
no only to 50 pages

**Comment by naruhina2016** [comment]
![Untitled](https://user-images.githubusercontent.com/18094007/69657885-71884800-107b-11ea-9625-c98d15077cb8.png)
sorry for double post but this what happened when i tried to download pic from page 59 on danbooru

**Comment by Nandaka** [comment]
ya, that's weird, because I can load just fine. Try to create new account and use it?

**Comment by naruhina2016** [comment]
for some reason sankaku behaves normally now but thanks anyways.

**Comment by hardnyasha** [comment]
i dont want to create a new topic, so i'll ask here @Nandaka . i've copied from request header my login and hash(sankaku), and now when trying to add a batch job, and everytime it says this:

![Снимок экрана (5)](https://user-images.githubusercontent.com/33638691/70321923-5a8ee600-1831-11ea-9b9e-2c53235a0077.png)

in the window above the contine and exit it says: key is empty or has empty meaning
[info.txt](https://github.com/Nandaka/DanbooruDownloader/files/3932098/info.txt)

![Снимок экрана (6)](https://user-images.githubusercontent.com/33638691/70322090-d0934d00-1831-11ea-897d-5287eabf9c22.png)

form provider looks like this. what did i do wrong?

**Comment by Nandaka** [comment]
can you restore the original DanbooruProviderList.xml from here: https://github.com/Nandaka/DanbooruDownloader/blob/master/DanbooruDownloader3/DanbooruProviderList.xml and add the batch job again?

**Comment by hardnyasha** [comment]
> can you restore the original DanbooruProviderList.xml from here: https://github.com/Nandaka/DanbooruDownloader/blob/master/DanbooruDownloader3/DanbooruProviderList.xml and add the batch job again?

updated. batch job works. lemme enter the hash again. entered, it works. i think it was because i didnt add a ";" at the end
